app/src/test/java/com/example/concordia_campus_guide/ModelTests/TrainTest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.concordia_campus_guide.ModelTests;
+
+import com.example.concordia_campus_guide.Models.Routes.Subway;
+import com.example.concordia_campus_guide.Models.Routes.Train;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class TrainTest {
+    private Train train;
+    private TestUtils testUtils;
+
+    @Before
+    public void init() {
+        testUtils = new TestUtils();
+    }
+
+    @Test
+    public void getColorTest() {
+        train = new Train(testUtils.getDirectionStepsObject());
+        assertEquals(testUtils.getDirectionStepsObject().transitDetails.line.shortName,train.getTrainShortName());
```

Comment:
Is this color the shortname? Can you specify this in a comment pls?

Reply:
We made a typo here thanks for catching it!